src/containers/InlinePlayer/InlinePlayer.tsx

```diff
@@ -16,22 +24,59 @@ type Props = {
   liveStartDateTime?: string | null;
   liveEndDateTime?: string | null;
   liveFromBeginning?: boolean;
+  startWatchingButton: React.ReactNode;
+  isLogged: boolean;
```

Comment:
`isLogged` -> `isLoggedIn`

But you can also obtain the `isLoggedIn` state within this container instead of passing it as prop.